Packs/Absolute/Integrations/Absolute/Absolute.py

```diff
@@ -0,0 +1,947 @@
+        parsed_device['CountryCode'] = geo_data.get('geoAddress', {}).get('countryCode')
+        parsed_device['Country'] = geo_data.get('geoAddress', {}).get('country')
+        parsed_device['ID'] = device.get('id')
+
+        parsed_response.append(parsed_device)
+
+    if len(parsed_response) == 1:
+        return parsed_response[0]
+    return parsed_response
+
+
+def get_device_application_list_command(args, client) -> CommandResults:
+    page = arg_to_number(args.get('page', 0))
+    limit = arg_to_number(args.get('limit', 50))
+
+    query_string = create_filter_query_from_args(args)
+    query_string = parse_return_fields(args.get('return_fields'), query_string)
+    query_string = parse_paging(page, limit, query_string)  # type: ignore
+
+    res = client.api_request_absolute('GET', '/v2/sw/deviceapplications', query_string=query_string)
+    if res:
+        outputs = parse_device_list_response(res)
+        human_readable = tableToMarkdown(f'{INTEGRATION} device applications list:', outputs, removeNull=True)
+        human_readable += f"Above results are with page number: {page} and with size: {limit}."
+        return CommandResults(outputs_prefix='Absolute.DeviceApplication', outputs=outputs, outputs_key_field='Appid',
+                              readable_output=human_readable, raw_response=res)
+    else:
+        return CommandResults(readable_output=f"No applications found in {INTEGRATION} for the given filters: {args}")
+
+
+def device_list_command(args, client) -> CommandResults:
+    page = arg_to_number(args.get('page', 0))
+    limit = arg_to_number(args.get('limit', 50))
+
+    query_string = create_filter_query_from_args(args, change_device_name_to_system=True)
+    query_string = parse_return_fields(",".join(DEVICE_LIST_RETURN_FIELDS), query_string)
+    query_string = parse_paging(page, limit, query_string)  # type: ignore
+
+    res = client.api_request_absolute('GET', '/v2/reporting/devices', query_string=query_string)
+    if res:
+        outputs = parse_device_list_response(copy.deepcopy(res), keep_os_in_list=False)
+        human_readable = tableToMarkdown(f'{INTEGRATION} devices list:', outputs, removeNull=True)
+        human_readable += f"Above results are with page number: {page} and with size: {limit}."
+        return CommandResults(outputs_prefix='Absolute.Device', outputs=outputs, outputs_key_field="Id",
+                              readable_output=human_readable, raw_response=res)
+    else:
+        return CommandResults(readable_output=f"No devices found in {INTEGRATION} for the given filters: {args}")
+
+
+def get_device_command(args, client) -> CommandResults:
+    if not ('device_ids' in args or 'device_names' in args or 'local_ips' in args or 'public_ips' in args):
+        raise DemistoException(
+            f"{INTEGRATION} at least one of the commands args (device_ids, device_names, local_ips, public_ips) "
+            f"must be provided.")
```

Comment:
Seems like this could use the generic replacement for `raise_error_on_missing_args`.